TesseractCore.xcodeproj/project.pbxproj

```diff
@@ -434,6 +446,7 @@
 				DYLIB_COMPATIBILITY_VERSION = 1;
 				DYLIB_CURRENT_VERSION = 1;
 				DYLIB_INSTALL_NAME_BASE = "@rpath";
+				FRAMEWORK_SEARCH_PATHS = "";
```

Comment:
Could you please delete this rather than setting it to the empty string? Selecting the row in Build Settings and hitting delete should do it.